### PR TITLE
NSF award app terraform changes_pm-view

### PR DESCRIPTION
**Pull request by vimalabhi9**
1. Almost all the changes done in pm-view folder except for adding the change in main gitignore to ignore the tfvars file from pm-view
2. Changes include the code to store all the relevant environment variables of pm-view to aws secret manager 
3. Updated code to limit few resource level permissions
4. Updated Redmine api URL, post the Redmine migration from ECS to EC2

### CODE REVIEW

File: pm_view/.env.default [hunk 1]
```diff
@@ -11,7 +11,7 @@ TF_VAR_auth_client_secret=
 TF_VAR_lod_roles_uri=
 TF_VAR_lod_programs_uri=
 TF_VAR_orcid_safelist=
-TF_VAR_redmine_api_url=http://redmine.ecs.redmine.default:3000
+TF_VAR_redmine_api_url=https://staging-redmine.bco-dmo.org
```
Comment (cschloer) [inline pm_view/.env.default:14]: Doesn't this need to be the private IP?

Comment (cschloer) [reply pm_view/.env.default:14]: I guess it's just in the env.default. Still might be useful to put the private IP here.

**Review by cschloer — APPROVED** [review]
Looks good - maybe just update the env.default file.

File: pm_view/.env.default (hunk 1 above)
Comment (vimalabhi9) [reply pm_view/.env.default:14]: Thanks. Will merge this and fix the url in env.default in next push

**Pull request merged**